weed/storage/needle/needle_read_write.go

```diff
@@ -278,6 +283,7 @@ func (n *Needle) readNeedleDataVersion2(bytes []byte) (err error) {
 		n.NameSize = uint8(bytes[index])
 		index = index + 1
 		if int(n.NameSize)+index > lenBytes {
+			stats.VolumeServerRequestCounter.WithLabelValues("errorIndexOutOfRange").Inc()
```

Comment:
Maybe it's better to move those strings such `errorIndexOutOfRange`, `errorCRC` etc. to a new block with constants to escape unnecessary repeats?

Same for `weed/topology` package

Reply:
Seems like a pretty rare event.
and more reason to go and see the logs and code

Reply:
> Maybe it's better to move those strings such `errorIndexOutOfRange`, `errorCRC` etc. to a new block with constants to escape unnecessary repeats?
> 
> 
> 
> Same for `weed/topology` package

Good suggestion! It'll be easier to have consistent metrics names also.